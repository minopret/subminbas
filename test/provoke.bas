90 LET B = 3 95 LET A = 5
100 PROGRAM PROVOKE_ERRORS
103 REM PROVOKE_ERRORS. PROVOKE EVERY SUBMINBAS SYNTAX ERROR, IN ORDER.
104 REM COPYRIGHT (C) 2020  AARON J. MANSHEIM
105 REM THE LICENSE FOR THIS PROGRAM IS GPL 3.0 OR LATER.
106 REM IT IS PROVIDED IN THE FILE: ../LICENSE
107 REM ALSO AT: HTTPS://WWW.GNU.ORG/%6C%69%63%65%6E%73%65%73/
110 PROVOKE EVERY POSSIBLE SYNTAX ERROR, IN ORDER
115 REM
116 REM `
117 REM
118 LET B = 3*
120 LET A = 2*(3-)
130 ON A GOTO A B 1, C
140 PRINT 3-+4
150 INPUT 1 2 A, 3
160 END
170 PRINT
180 INPUT A
