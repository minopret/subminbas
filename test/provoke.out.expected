[error 1] line 90 near "95": syntax error
line 90 near "95": statement: error
line 90 near "95": statement: error
line 90 near " LET": statement: error
line 90 near "A": statement: error
line 90 near "=": statement: error
line 90 near "5": statement: error
line 100 near " PROGRAM": statement: error
line 100 near " PROGRAM": statement: error
line 100 near "P": statement: error
line 100 near "R": statement: error
line 100 near "O": statement: error
line 100 near "V": statement: error
line 100 near "O": statement: error
line 100 near "K": statement: error
line 100 near "E": statement: error
line 100 near "_": statement: error
line 100 near "E": statement: error
line 100 near "R": statement: error
line 100 near "R": statement: error
line 100 near "O": statement: error
line 100 near "R": statement: error
line 100 near "S": statement: error
[error 2] line 110 near " PROVOKE": syntax error
line 110 near " PROVOKE": statement: error
line 110 near " PROVOKE": statement: error
line 110 near " EVERY": statement: error
line 110 near " POSSIBLE": statement: error
line 110 near " SYNTAX": statement: error
line 110 near "E": statement: error
line 110 near "R": statement: error
line 110 near "R": statement: error
line 110 near "O": statement: error
line 110 near "R": statement: error
line 110 near ",": statement: error
line 110 near " IN": statement: error
line 110 near " ORDER": statement: error
[error 3] line 116 near "`": syntax error
line 116 near "`": statement: error
line 116 near "`": statement: error
[error 4] line 118: syntax error
line 118: statement: error
[error 5] line 120 near ")": syntax error
line 120 near ")": numeric_expression: '(' error ')'
[error 6] line 130 near "A": syntax error
line 130 near "A": line_number_ref_list: error
line 130 near "B": line_number_ref_list: line_number_ref_list error
line 130 near "1": line_number_ref_list: line_number_ref_list error line_number_ref
[error 7] line 130 near "C": syntax error
line 130 near "C": line_number_ref_list: line_number_ref_list ',' error
line 130: line_number_ref_list: line_number_ref_list error
[error 8] line 140 near "+": syntax error
line 140 near "+": print_list: print_list error
line 140 near "+": print_list: print_list error
[error 9] line 150 near "1": syntax error
line 150 near "1": variable_list: error
line 150 near "2": variable_list: variable_list error
line 150 near "A": variable_list: variable_list error variable
[error 10] line 150 near "3": syntax error
line 150 near "3": variable_list: variable_list ',' error
line 150: variable_list: variable_list error
[error 11] line 170 near "170": syntax error
line 170 near "170": program: blocks end_line error
line 170 near "170": blocks: error
line 170: blocks: blocks error
line 180 near "180": blocks: blocks error
[error 12] line 180: syntax error
line 180: program: blocks error
